docs/migrations/v0.15.md

```diff
@@ -0,0 +1,28 @@
+# v0.15
```

Comment:
Is this document intended to be explaining a migration from a version **before** v0.15.x to v0.15.x? 
Asking this as it might generate some confusion on developers which are in versions before v0.14.

Reply:
Good point, I think you can migrate from a few versions back without any major impact, but I'll call out that this doc is for going from v0.14.x to v0.15